assets/ll_default_config.json

```diff
@@ -0,0 +1,114 @@
+{
+"project": {
+	"chanlocs": "GSN-HydroCel-129",
```

Comment:
Do you think we should eventually split the proect_config portion into a separate YAML file?

Reply:
I don't think so. It was in a separate config and I merged it purposefully. I think there is a lot of refactoring to be done regarding these config parameters, and we can consider down the line splitting it... but that would require a clear use case (e.g., splitting project-independent from project-dependent config), but as far as I can tell, everything in that config file is project-dependent / project-specific so I currently do not see any up side and I see down side (having bits and pieces of project config in separate files creates confusion) in splitting it. This is currently the one and only file containing project-specific information that the subject would need to provide to run the pipeline. But feel free to share your thoughts if you see reasons to split it.